index.js

```diff
@@ -4,15 +4,42 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = 'green'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const profile = document.querySelector('#profile')
+const nameData = profile.querySelector('h2')
+nameData.textContent = traveler.name
+
+const nickname = profile.querySelector('em')
+nickname.textContent = traveler.nickname
 
+const image = profile.querySelector('img')
+image.src = traveler.photo
+image.alt = traveler.name
```

Comment:
Nice work scoping the search to the `profile`. Scoping our searches is also better performance wise because there's less DOM traversal needed to be done!